modules/ROOT/pages/sharepoint/sharepoint-connector.adoc

```diff
@@ -2,39 +2,68 @@
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
+:imagesdir: ../../assets/images/
 
-Support Category: Select
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
 
-The Microsoft SharePoint connector provides support for both SharePoint 2013 and SharePoint Online for use in the cloud and on-premises. This connector enables content and document management, access to intranet and extranet portals and websites, collaboration, and enterprise search. This connector also enables integration with SharePoint through its REST API.
+Anypoint Connector for Microsoft SharePoint (Microsoft SharePoint Connector) version 3.1
+
+Release Notes: xref:release-notes::connector/sharepoint-connector-release-notes-mule-4.adoc[Microsoft SharePoint Connector Release Notes - Mule 4]
+
+Microsoft SharePoint Connector provides support for both SharePoint 2013 and SharePoint Online for use in the cloud and on-premises. This connector enables content and document management, access to intranet and extranet portals and websites, collaboration, and enterprise search. This connector also enables integration with SharePoint through its REST API.
 
 == Prerequisites
 
-This document assumes that you are familiar with SharePoint, Mule, Anypoint Connectors, Anypoint Studio, Mule concepts, elements in a Mule flow, and Global Elements.
+This document assumes that you are familiar with Microsoft SharePoint, Mule, Anypoint Connectors, Anypoint Studio, Mule concepts, elements in a Mule flow, and Global Elements.
 
 You need login credentials to test your connection to your target resource.
 
 For hardware and software requirements and compatibility
-information, see the Connector Release Notes.
+information, see the connector Release Notes.
 
-To use this connector with Maven, view the pom.xml dependency information in
-the Dependency Snippets in Anypoint Exchange.
+To use this connector with Apache Maven, view the pom.xml dependency information in
+the Dependency Snippets in https://www.mulesoft.com/exchange/com.mulesoft.connectors/mule-sharepoint-connector/[Anypoint Exchange].
 
 == What's New in this Connector
 
-In Mule 4 SharePoint Online and SharePoint OnPremise were merged into one connector that includes all functionalities from both previous connectors.
+In Mule 4 SharePoint online and SharePoint on-premises were merged into one connector that includes all functionalities from both previous connectors.
+
+== Connect in Design Center
+
+Anypoint Design Center's Flow Designer enables you to create apps visually. For more information, see the xref:design-center::fd-tour.adoc[Flow Designer Tour].
+
+To create an app in Flow Designer:
+
+* Configure the input source (trigger) for your app.
+* Configure the target component for your app.
```

Comment:
Are these supposed to be chronological, or does order not matter?

Reply:
GLOBAL:
Standardize between the more precise of "input source (trigger)" and "input source trigger."
These two texts have DIFFERENT meanings: one implies synonyms and the other a type of trigger.